pkg/apis/monitoring/v1/types.go

```diff
@@ -123,6 +123,9 @@ type PrometheusSpec struct {
 	// The labels to add to any time series or alerts when communicating with
 	// external systems (federation, remote storage, Alertmanager).
 	ExternalLabels map[string]string `json:"externalLabels,omitempty"`
+	// Enable access to prometheus web admin API. Defaults to the value of `true`.
+	// For more information see https://prometheus.io/docs/prometheus/latest/querying/api/#tsdb-admin-apis
+	EnableAdminApi string `json:"enableAdminApi,omitempty"`
```

Comment:
this should be `EnabledAdminAPI` and the json string should then be `enableAdminAPI`

Reply:
Enabled is past tense and it is better to have `Enable` in this case. `enableAdminApi` is the flag I have used elsewhere in code as well. Let me know what you think

Reply:
Whops sorry that was a typo on my end. I meant `EnableAdminAPI` for the struct field and `enableAdminAPI` for the json field.

Reply:
Got it, good catch :)